ecommerce/serializers.py

```diff
@@ -955,7 +958,9 @@ def get_coupon(self, instance):
 
     def get_purchaser(self, instance):
         """Get the purchaser infrmation"""
-        return ExtendedLegalAddressSerializer(instance.purchaser.legal_address).data
+        data = ExtendedLegalAddressSerializer(instance.purchaser.legal_address).data
+        data.update({"company_name": instance.purchaser.profile.company})
```

Comment:
I did something like that 
https://github.com/mitodl/mitxpro/pull/2125/files#diff-4349bb2be4e3bfa15b93d2cf390f4df197e2a0b11e6edfe8abd3f09beec21273R159-R177